core/templates/dev/head/pages/exploration-editor-page/exploration-editor-page.controller.ts

```diff
@@ -260,465 +260,485 @@ require('services/StateTopAnswersStatsService.ts');
 require('pages/exploration-editor-page/exploration-editor-page.constants.ts');
 require('pages/interaction-specs.constants.ts');
 
-oppia.controller('ExplorationEditor', [
```

Comment:
The file nmae is `....controller.ts` whereas we are creating a directive. is this an expected structure?

Reply:
Yes @DubeySandeep The files when renamed and modified (>50%) at the same time show incorrect diffs of add/delete. Therefore I have only modified it here and will rename it in a follow up PR.